<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>newMap</class>
 <widget class="QWidget" name="newMap">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>346</width>
    <height>193</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>New Map</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>151</width>
     <height>181</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Format:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QListWidget" name="listWidget">
      <property name="editTriggers">
       <set>QAbstractItemView::SelectedClicked</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>10</y>
     <width>171</width>
     <height>101</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Layers:</string>
      </property>
     </widget>
    </item>
    <item alignment="Qt::AlignTop">
     <widget class="QSlider" name="horizontalSlider">
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>4</number>
      </property>
      <property name="value">
       <number>2</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="tickPosition">
       <enum>QSlider::TicksBothSides</enum>
      </property>
      <property name="tickInterval">
       <number>0</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Height / Weight (INT)</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="sizeEdit">
      <property name="inputMask">
       <string notr="true"/>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="maxLength">
       <number>32767</number>
      </property>
      <property name="frame">
       <bool>true</bool>
      </property>
      <property name="cursorPosition">
       <number>0</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>110</y>
     <width>171</width>
     <height>81</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Name:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="nameEdit">
      <property name="maxLength">
       <number>15</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QDialogButtonBox" name="saveBox">
      <property name="standardButtons">
       <set>QDialogButtonBox::Cancel|QDialogButtonBox::Save</set>
      </property>
      <property name="centerButtons">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
